tests/tests.sh

```diff
@@ -32,7 +32,7 @@ function dev_setup {
 }
 
 function run_tests {
-  run_playbook "$DIR/run.yml" -e streisand_ci=true "$1"
+  run_playbook "$DIR/run.yml" --extra-vars=@"$DIR/vars_ci.yml" "$1"
```

Comment:
Can you explain this diff?

Reply:
@cpu I would like to pass more vars to `run.yml` and enumerating them here looks ugly. So I put all (three) vars in a file and pass the file.

Reply:
I think you can just set these in `test/group_vars/all/all`. We separate the vars for CI so that it's easier to add things to satisfy test prompts.

Reply:
@cpu No problems. I have moved CI variables from extra- to group_vars.

Reply:
@cpu Oh no. My memory tricked me. I remebered that why I move the CI variables from group_vars to extra vars: Defining them in group_vars does not provide enough priority so Ansible will still asks questions: https://travis-ci.org/jlund/streisand/builds/271395890

I'll delete the commit.